main.py

```diff
@@ -13,9 +14,18 @@ def endings():
     chosen_ending = random.choice(ending_list)
     print(chosen_ending)
 
-def start():
-    start1 = ('''
-You wake up in a forest 
+def question_name():
```

Comment:
Do you need to ask for the user's name? I thought you assigned a character to the user.

Reply:
The story is going to play around the person who's playing. So whatever they input as their name will be the name used throughout the story